code/modules/clothing/clothing.dm

```diff
@@ -312,7 +312,6 @@
 
 	var/list/modes = list("Off", "Binary sensors", "Vitals tracker", "Tracking beacon")
 	var/switchMode = input("Select a sensor mode:", "Suit Sensor Mode", modes[sensor_mode + 1]) in modes
```

Comment:
Did you want to change this one? you only removed a line here

Reply:
Yes, I did. But it didn't work out so well, because you can't set the default button in the tgui list, so I decided not to touch it.
Maybe someday I'll be able to add this function.